README.md

```diff
@@ -57,33 +58,67 @@ To use this library in your project manually you may:
 1. for Projects, just drag BackgroundVideo.swift to the project tree
```

Comment:
This should be changed to reflect our name change. `BackgroundVideo.swift` -> `VideoBackground.swift`

Reply:
@quanvo87 here is the typo